<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmNMRFSpectrum</class>
 <widget class="QMainWindow" name="FrmNMRFSpectrum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NMR Spectrum Averager (Frequency Sweep)</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="m_ckbActive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Acti&amp;vate Freq. Sweep</string>
        </property>
        <property name="shortcut">
         <string>Alt+V</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Center Freq</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_edCenterFreq">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>MHz</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Span</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_edFreqSpan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>kHz</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Step</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_edFreqStep">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>kHz</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Band Width</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_edBW">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>kHz</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="textLabel1_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pulse Analyzer</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbPulse"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox1">
        <property name="title">
         <string>Signal Generator</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="m_cmbSG1"/>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Offset Freq.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLineEdit" name="m_edSG1FreqOffset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textLabel2_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>MHz</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox1_3">
        <property name="title">
         <string>Burst Mode</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="textLabel1_3_3_2_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Count</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_numBurstCount"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="KUrlRequester" name="m_urlDump">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnDump">
          <property name="text">
           <string>D&amp;UMP</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="XQGraph" name="m_graph" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox3">
        <property name="title">
         <string>Post Processing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel1_2_2_3_2_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Solver</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="m_cmbSolver">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="m_ckbAutoPhase">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Auto Phase</string>
             </property>
             <property name="shortcut">
              <string>Alt+A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KDoubleNumInput" name="m_numPhase">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string>deg.</string>
             </property>
             <property name="minValue" stdset="0">
              <number>-360</number>
             </property>
             <property name="maxValue" stdset="0">
              <number>360</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="textLabel11_4_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Window Func.</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="m_cmbWindowFunc"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="textLabel1_2_2_3_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Width</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KDoubleNumInput" name="m_numWindowWidth">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="referencePoint">
                <double>10.000000000000000</double>
               </property>
               <property name="relativeValue">
                <double>1.000000000000000</double>
               </property>
               <property name="minValue" stdset="0">
                <number>10</number>
               </property>
               <property name="maxValue" stdset="0">
                <number>200</number>
               </property>
               <property name="precision" stdset="0">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="m_btnClear">
             <property name="text">
              <string>&amp;CLEAR SPECTRUM</string>
             </property>
             <property name="shortcut">
              <string>Alt+C</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel1_2_2_3_2_2_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Band Width</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="m_cmbBWList">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>m_ckbActive</zorder>
   <zorder>groupBox1_3</zorder>
   <zorder>groupBox1</zorder>
   <zorder>groupBox3</zorder>
   <zorder>textLabel1_4</zorder>
   <zorder>groupBox3</zorder>
   <zorder>m_edFreqStep</zorder>
   <zorder>groupBox1</zorder>
   <zorder>groupBox1_3</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="4" margin="2"/>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>XQGraph</class>
   <extends>QWidget</extends>
   <header>graphwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_ckbActive</tabstop>
  <tabstop>m_edCenterFreq</tabstop>
  <tabstop>m_edFreqSpan</tabstop>
  <tabstop>m_edFreqStep</tabstop>
  <tabstop>m_edBW</tabstop>
  <tabstop>m_cmbPulse</tabstop>
  <tabstop>m_cmbSG1</tabstop>
  <tabstop>m_edSG1FreqOffset</tabstop>
  <tabstop>m_numBurstCount</tabstop>
  <tabstop>m_urlDump</tabstop>
  <tabstop>m_btnDump</tabstop>
  <tabstop>m_cmbSolver</tabstop>
  <tabstop>m_ckbAutoPhase</tabstop>
  <tabstop>m_cmbWindowFunc</tabstop>
  <tabstop>m_btnClear</tabstop>
  <tabstop>m_cmbBWList</tabstop>
 </tabstops>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
